src/pages/View/ChartView/ChartView.js

```diff
@@ -66,89 +60,79 @@ export default class ChartView extends Component {
         this.setState({note: e.target.value})
     }
 
-    //Not currently being used
-    /*
-    openModal = () => {
-       this.setState({modalIsOpen: true});
-     }
-     */
-
     closeModal = () => {
        this.setState({modalIsOpen: false});
      }
 
     render() {
         const {open} = this.state;
 
+        let rows = []
```

Comment:
I'm assuming it doesn't really matter, but maybe it would be cooler to add a semicolon after let rows = [];
And potentially in various other places below? *shrugs casually